### PR TITLE
highlighting new snippets in UI

### PR DESCRIPTION
**Pull request by Jbarget**

### CODE REVIEW

File: client/src/Model/Model.elm [hunk 1]
```diff
@@ -150,9 +153,15 @@ type alias Model =
     , notifications : Notifications
     , screenData : ScreenData
     , snippetBookmarksDetail : SnippetBookmarksDetail
+    , currentDate : Maybe Date
     }
 
 
+getCurrentDate : Cmd Msg
+getCurrentDate =
+    Task.perform (Just >> SetDate) Date.now
```
Comment (mantagen) [inline client/src/Model/Model.elm]: can we put this in Helpers/CmdMsg.elm

File: client/src/Update/Update.elm [hunk 2]
```diff
@@ -311,16 +312,16 @@ update msg model =
                 queryCmd =
                     case ( newModel.location.hash, queryValidation updatedModel ) of
                         ( "#/query", Valid ) ->
-                            QueryDecoder.requestCmd newModel Nothing
+                            [ QueryDecoder.requestCmd newModel Nothing, Model.getCurrentDate ]
```
Comment (mantagen) [inline client/src/Update/Update.elm]: can we intead of storing a stale DateTime, have "getCurrentDate" run every second to keep it up to date?

Comment (mantagen) [reply client/src/Update/Update.elm]: -- alternatively, store it as `latestQueryTime`? ;)

Comment (Jbarget) [reply client/src/Update/Update.elm]: i think until we have a need to keep an updated Date and time lets leave it how it is, will adjust the name

File: client/src/Model/Model.elm (hunk 1 above)
Comment (Jbarget) [reply client/src/Model/Model.elm]: circular dependency :(

Comment (mantagen) [reply client/src/Model/Model.elm]: what if you make function?
```
getCurrentDate : msg -> Cmd cmd
getCurrentDate msg =
    Task.perform (Just >> msg) Date.now
```
cause the problem would be importing from model?

then call it `getCurrentDate SetDate` (callback styleee)

Comment (Jbarget) [reply client/src/Model/Model.elm]: yah good shout, tried it but it didnt like it :(

**Comment by Jbarget** [comment]
screen shot 2018-09-07 at 15 36 28

**Pull request merged**